docs/cloudinit.md

```diff
@@ -18,25 +16,27 @@ So it means very easily customizing your VM when you create it from a compatible
 
 You only need to use a template of a VM with CloudInit installed inside it. [Check this blog post to learn how to install CloudInit](https://xen-orchestra.com/blog/centos-cloud-template-for-xenserver/).
 
+**Note:** In XOA 5.31, we changed the cloud-init config drive type from [OpenStack](https://cloudinit.readthedocs.io/en/latest/topics/datasources/configdrive.html) to the [NoCloud](https://cloudinit.readthedocs.io/en/latest/topics/datasources/nocloud.html) type. This will allow us to pass network configuration to VMs in the future. For 99% of users, including default cloud-init installs, this change will have no effect. However if you have previously modified your cloud-init installation in a VM template to only look for `openstack` drive types (for instance with the `datasource_list` setting in `/etc/cloud/cloud.cfg`) you need to modify it to also look for `nocloud`.
+
 ## Usage
 
 First, select your compatible template (CloudInit ready) and name it:
 
-![](https://xen-orchestra.com/blog/content/images/2015/12/template_choice.png)
+![](https://i.imgur.com/XDAy7DB.png)
```

Comment:
Is it allowed to directly use an picture from `imgur`?

Can they change their policy and block them?

Reply:
it's allowed to directly use them. I doubt they would change their policy but they could disappear. I would prefer to host them on xen-orchestra.com but I do not have permission/access to do so. If you can mirror those PNGs on our domain and stick them in there that would probably be better

Reply:
Why not store them in `docs/assets` and use them directly? :slightly_smiling_face:

Reply:
now you're cooking with gas

Reply:
fixed